docs/installation/sources/sources_mac.rst

```diff
@@ -0,0 +1,370 @@
+       sudo cmake --build . --target install
+
+.. note::
+
+    By default, *eProsima Fast DDS* does not compile tests.
+    However, they can be activated by downloading and installing `Gtest <https://github.com/google/googletest>`_.
+
+
+.. _global_installation_sm:
+
+Global installation
+^^^^^^^^^^^^^^^^^^^
+
+To install *eProsima Fast DDS* system-wide instead of locally, remove all the flags that
+appear in the configuration steps of :code:`Fast-CDR` and :code:`Fast-DDS`, and change the first in the
+configuration step of :code:`foonathan_memory_vendor` to the following:
+
+.. code-block:: bash
+
+    -DCMAKE_INSTALL_PREFIX=/usr/local/ -DBUILD_SHARED_LIBS=ON
+
+
+.. _run_app_cmake_sm:
+
+Run an application
+^^^^^^^^^^^^^^^^^^
+
+When running an instance of an application using *eProsima Fast DDS*, it must be linked with the library where the
+packages have been installed, which in the case of system-wide installation  is: :code:`/usr/local/lib/` (if local
+installation is used, adjust for the correct directory).
+There are two possibilities:
+
+* Prepare the environment locally by typing the command:
+
+  .. code-block:: bash
+
+      export LD_LIBRARY_PATH=/usr/local/lib/
+
+* Add it permanently it to the :code:`PATH`, by typing:
+
+  .. code-block:: bash
+
+      touch ~/.bash_profile
+      echo 'export LD_LIBRARY_PATH=/usr/local/lib/' >> ~/.bash_profile
+
+
+.. _fastddsgen_sm:
+
+Fast DDS-Gen installation
+"""""""""""""""""""""""""
+
+This section provides the instructions for installing *Fast DDS-Gen* in a Mac OS environment from
+sources.
+*Fast DDS-Gen* is a Java application that generates source code using the data types defined in an IDL file.
+Please refer to :ref:`fastddsgen_intro` for more information.
+
+Requirements
+------------
+
+In order to compile *Fast DDS-Gen*, the following packages need to be installed in the system:
+
+* :ref:`java_sm`
+* :ref:`gradle_sm`
+
+.. _java_sm:
+
+Java JDK
+^^^^^^^^
+
+The JDK is a development environment for building applications and components using the Java language.
+Download and install it at the following the steps given in the
+`Oracle website <https://www.oracle.com/java/technologies/javase-downloads.html>`_.
+
+.. _gradle_sm:
+
+Gradle
+^^^^^^
+
+Gradle is an open-source build automation tool.
+Download and install the last stable version of `Gradle <https://gradle.org/install>`_ in the preferred way.
+with Homebrew it would be running the command:
+
+.. code-block:: bash
+
+        brew install gradle
+
+
+Compiling Fast DDS-Gen
+----------------------
+
+Once the requirements above are met, compile *Fast DDS-Gen* by following the steps below:
+
+.. code-block:: bash
+
+    cd ~
+    git clone --recursive https://github.com/eProsima/Fast-DDS-Gen.git
+    cd Fast-DDS-Gen
+    gradle assemble
+
+Contents
+^^^^^^^^
+
+The :code:`Fast-DDS-Gen` folder contains the following packages:
+
+* :code:`share/fastrtps`, where the generated Java application is.
```

Comment:
```suggestion
* :code:`share/fastddsgen`, where the generated Java application is.
```
It should be great if this could be updated in the other installation instructions